Update README to link to Wiki for options + API documentation

nmielnik opened the pull request:
There is now more detailed and thorough documentation in place via the Wiki, which is a superset of the details in the README.  I was thinking it might be better to have a shorter README and provide links to the pages which have documentation. 

Do you all agree or do you think it's better to have a longer, one-stop-shop style README which include detailed documentation and which is maintained via source control?

j0k3r commented:
I like to keep doc in the repo since it's easier to maintain an keep tracks on what change related to which PR. I mean, It's hard to track change in the repo which can affect the wiki.

Wiki on Github is great, but I think we should have a proper markdown documentation. I know there is [Couscous](http://couscous.io/) for PHP project but it might have the same kind of documentation builder for JS project.

phiggins42 commented:
I like keeping docs as close to the source as possible.Wikis are great for tutorials and longwinded examples, but they so often fall stale. A proper auto-generated MD tree (or a manually maintained for that matter) in-source way easier to maintain and track changes on.

nmielnik commented:
Ok, makes sense to me.  I've added the documentation on the options and API methods back into the README.  

I've added some updates to the buttons documentation in the README, as well as rearranged the list of API methods (as well as added 1 or 2 that were missing).
